ocamldoc/odoc_args.ml

```diff
@@ -137,6 +137,7 @@ let analyse_merge_options s =
     (M.merge_raised_exception, [Odoc_types.Merge_raised_exception]) ;
     (M.merge_return_value, [Odoc_types.Merge_return_value]) ;
     (M.merge_custom, [Odoc_types.Merge_custom]) ;
+    (M.merge_alert, [Odoc_types.Merge_alert]) ;
```

Comment:
Thinking a bit about this point, this is not the right semantic for alerts: alerts from the implementation are never merged nor lifted into the interface. This is not an user controlled behavior on the compiler side and thus it should not be an option on the ocamldoc side.

Reply:
Unrelated to this PR but isn't it weird that documentation comments aren't hidden by an interface file ? The generated documentation has more content than what's in the `mli` file and some documentation might be incomplete when switching from ocamldoc to odoc, which doesn't look at the implementation if it founds an interface.

Reply:
This is part of the behavior of ocamldoc that was intentionally not reproduced in odoc, because merging information from two slightly incompatible different sources can be really confusing (as illustrated by the `@param` tag).

Reply:
I tweaked the merge but a bug remain: The deprecated alert is converted to a deprecated tag too early and will be merged anyway.
This is was on purpose initially but I'm happy to change that if you think we should. It would be a big change though.

Reply:
No, that's enough time spent on ocamldoc. At this point, it is more a bug on the ocamldoc implementation of deprecation tags.